src/www/ui/admin-license-file.php

```diff
@@ -57,6 +57,7 @@ function RegisterMenus()
   public function Output()
   {
     $V = ""; // menu_to_1html(menu_find($this->Name, $MenuDepth),0);
+    $V .= '<script src="scripts/jquery-3.2.0.min.js"></script>';
```

Comment:
I think for this problem there is the usual [`$(document).ready()`-pattern](https://learn.jquery.com/using-jquery-core/document-ready/). Just call your function to populate the table after everything is loaded.